[2025-01-27T17:59:39.008+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-27T17:59:39.044+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cross_task_communication.increment_by_1 manual__2025-01-27T17:59:36.846106+00:00 [queued]>
[2025-01-27T17:59:39.062+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cross_task_communication.increment_by_1 manual__2025-01-27T17:59:36.846106+00:00 [queued]>
[2025-01-27T17:59:39.066+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-27T17:59:39.094+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): increment_by_1> on 2025-01-27 17:59:36.846106+00:00
[2025-01-27T17:59:39.111+0000] {standard_task_runner.py:72} INFO - Started process 4286 to run task
[2025-01-27T17:59:39.115+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cross_task_communication', 'increment_by_1', 'manual__2025-01-27T17:59:36.846106+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/cross_task_communicat.py', '--cfg-path', '/tmp/tmpkuq2bsie']
[2025-01-27T17:59:39.118+0000] {standard_task_runner.py:105} INFO - Job 45: Subtask increment_by_1
[2025-01-27T17:59:39.218+0000] {task_command.py:467} INFO - Running <TaskInstance: cross_task_communication.increment_by_1 manual__2025-01-27T17:59:36.846106+00:00 [running]> on host 6eaa7309f7d4
[2025-01-27T17:59:39.379+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cross_task_communication' AIRFLOW_CTX_TASK_ID='increment_by_1' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T17:59:36.846106+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-27T17:59:36.846106+00:00'
[2025-01-27T17:59:39.385+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-27T17:59:39.407+0000] {logging_mixin.py:190} INFO - Count 100!
[2025-01-27T17:59:39.410+0000] {python.py:240} INFO - Done. Returned value was: 101
[2025-01-27T17:59:39.447+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-27T17:59:39.449+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cross_task_communication, task_id=increment_by_1, run_id=manual__2025-01-27T17:59:36.846106+00:00, execution_date=20250127T175936, start_date=20250127T175939, end_date=20250127T175939
[2025-01-27T17:59:39.496+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-27T17:59:39.534+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-27T17:59:39.542+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
